portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PageProfilePath.testcase

```diff
@@ -137,4 +137,58 @@ definition {
 		}
 	}
 
+	@description = "Bug: LRAC-8988 | Automation ID: LRAC-11059 | Test Summary: Create the page name with special characters makes the correct name appear in the path part of AC pages"
+	@priority = "3"
+	test CreatePageNameWithSpecialCharactersMakeCorrectNameAppearInPath {
+		task ("Create a site and a blog and add to a page") {
+			ACUtils.addSiteAndPage();
```

Comment:
Hey @manuferreira, I think you should update tearDown to delete the site you created so it doesn't affect other tests in sequence:

`else {
	 JSONGroup.deleteGroupByName(groupName = "Site Name");
}`


What do you think?